code/datums/components/pellet_cloud.dm

```diff
@@ -110,6 +110,7 @@
 	var/original_damage = shell.loaded_projectile.damage
 	var/original_wb = shell.loaded_projectile.wound_bonus
 	var/original_bwb = shell.loaded_projectile.bare_wound_bonus
+	var/original_if = shell.loaded_projectile.ignored_factions
```

Comment:
Could you just change the variable name to ``original_ignored_factions``, and do the same for the variables above it? It isn't clear what these variables mean in isolation, and we much prefer clarity over shorthand in most instances. 

If you spot any single letter variables, we would appreciate you doing the same for those as well. Since that runs into the same problem.

Reply:
updated.